### ISSUE TITLE
Invisible footer link after clicking into a new window

### ISSUE THREAD

**Issue by jimleroyer** [cds-snc/notification-planning#1911]
## Describe the bug 

The text color matches the background color when opening a link from the menu footer in a new window.

<img width="638" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBeUlCQ2c9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--5abfc3e865a1322602c2a273659f73f183bf738d/image.png" alt="image.png" />

The text for New Features link on the screenshot above is not visible anymore, although the underline is present because it has been clicked. This is how it looks prior to the click:

<img width="562" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBeVFCQ2c9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--54e31a35988057f0497eb311aafff3c206c6f850/image.png" alt="image.png" />

## [Bug Severity](https://docs.google.com/document/d/1tcD32bjJRaREsMgaUEoXAk400y1dKzzaKR0G5Sr1w20/edit)

See examples in the documentation

(SEV-1 Critical, SEV-2 Major, SEV-3 Minor, SEV-4 Low) 

## To Reproduce

1. Go to the https://notification.canada.ca/ address.
2. Click on the *New Features* link in the footer menu and maintain the CMD button (to open in a new window).

The text color of the link should now be of the same color as the background, making it invisible.

## Expected behavior

The link text should be visible in a color different from the background color.

## Impact

If applicable

Impact on Notify users:
Accessibility issue for users relying on visual text.

Impact on Recipients:
None

Impact on Notify team: 
None

## Screenshots

<img width="638" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBeXNCQ2c9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--9380a7ad6bef6c60445a541347816be7306d19d9/image.png" alt="image.png" />

## Additional context

None

**Comment by yaelberger-commits** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Notify-Planning-614b3ad91bc2030015ed22f5/issues/gh/cds-snc/notification-planning/1911?planning-poker) with Zenhub @amazingphilippe @andrewleith

**Comment by YedidaZalik** [comment]
Put up PR yesterday
Removing styles that don't do anything but break links

**Comment by amazingphilippe** [comment]
A fix is ready: https://github.com/cds-snc/notification-admin/pull/2183

**Comment by jimleroyer** [comment]
Thank you team! 🙇 I did see the fix deployed in production last Friday and appreciate the timely fix!
